### ISSUE TITLE
Android Import Issue!

### ISSUE THREAD

**Issue by rsflow** [realm/realm-js#319]
Hi,

When the app is turned off and it receives a push notification `import Realm` is called and this error appears:
![screen shot 2016-03-09 at 6 29 33 pm](https://cloud.githubusercontent.com/assets/6751025/13644230/efd6ae66-e624-11e5-8ade-1115eb721f3c.png)
The same issue happened with CodePush by Microsoft but it was fixed by calling the constructor only when a specific method was called (ex. `CodePush.sync()`).

Thank You!

**Comment by alazier** [comment]
Is this on android or iOS? We will try to reproduce the issue on our side. If you have a simple application that we can use to reproduce it would be extremely helpful.

**Comment by DHidee66** [comment]
Hey @alazier ,

So when the app is turned off the top `if` statement is executed and also all the `import` statements are called and Realm gets initialized.

```
import React, { View, AppRegistry, Component, DeviceEventEmitter } from 'react-native'
import CodePush from 'react-native-code-push'
import Realm from 'realm'
import GCMAndroid from 'react-native-gcm-android'
import Notification from 'react-native-system-notification'

import App from './src/app'

if (GCMAndroid.launchNotification) {

    Notification.create(...)
    GCMAndroid.stopService()

} else {

  class App extends Component {
    componentDidMount() {
      CodePush.sync()

      GCMAndroid.addEventListener('notification', (notification) => this.showNotification(notification))
      GCMAndroid.requestPermissions()
    }
    render() {
      return <App />
    }
  }

  AppRegistry.registerComponent('App', () => App)
}
```

**Comment by alazier** [comment]
Are you still having this issue? If so can you try with the new release to see if things are resolved?

**Comment by appden** [comment]
@rsflow We just released version [0.11.0](https://github.com/realm/realm-js/releases/tag/v0.11.0), which _might_ resolve this issue. Would you please confirm if this issue is still occurring after updating? Thanks!

**Comment by rsflow** [comment]
Hey guys as soon as I try it I will let you know. Thank you :)
